### PR TITLE
navbar / footer refactor and added formatter

### PR DESCRIPTION
**Pull request by RandolphTang**
the whole nav / footer files and CSS is refactored. Added formatter

### CODE REVIEW

File: src/Components/Navbar.js [hunk 1]
```diff
@@ -103,21 +94,36 @@ function Navbar() {
             tooltip="Home"
             label="Home"
           />
+          {/*<NavItem*/}
+          {/*  icon={<FcViewDetails />}*/}
+          {/*  route={"/About"}*/}
```
Comment (elimelt) [inline src/Components/Navbar.js:99]: prob shouldn't leave commented

**Review by elimelt — APPROVED** [review]
LGTM after commented out code removed

File: src/Components/Navbar.js [hunk 2]
```diff
@@ -75,24 +77,13 @@ function Navbar() {
...
-        }
-        route={"/"}
-        closeExpand={closeExpand}
-      />
-
-      <NavToggle icon={<ImSun />} />
-      <Switch toggleMode={toggleMode} />
-      <NavToggle icon={<PiMoonBold />} />
+      <li className={`nav-Logo-Item`} onClick={closeExpand}>
+        <NavLink to={"/"}>
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:83]: doesn't need `{` or `}`

File: src/Components/Navbar.js [hunk 3]
```diff
@@ -75,24 +77,13 @@ function Navbar() {
...
-          </NavLink>
-        }
-        route={"/"}
-        closeExpand={closeExpand}
-      />
-
-      <NavToggle icon={<ImSun />} />
-      <Switch toggleMode={toggleMode} />
-      <NavToggle icon={<PiMoonBold />} />
+      <li className={`nav-Logo-Item`} onClick={closeExpand}>
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:82]: I don't think the nav-Logo-item needs `{` or `}`

File: src/Components/Navbar.js [hunk 4]
```diff
@@ -103,21 +94,36 @@ function Navbar() {
...
+          {/*  icon={<FcViewDetails />}*/}
+          {/*  route={"/About"}*/}
+          {/*  closeExpand={closeExpand}*/}
+          {/*  tooltip="About"*/}
+          {/*  label="About"*/}
+          {/*/>*/}
           <NavItem
             icon={<FcConferenceCall />}
-            route={"/About"}
+            route={"/Officers"}
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:106]: doesn't need `{` or `}`

**Review by ShawnCollinge — CHANGES_REQUESTED** [review]

File: src/Components/Navbar.js [hunk 5]
```diff
@@ -103,21 +94,36 @@ function Navbar() {
...
             expand={expand}
             closeExpand={closeExpand}
             tooltip="Officers"
             label="Officers"
           />
+          <NavItem
+            icon={<FaHeart />}
+            route={"/Sponsor"}
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:114]: doesn't need the brackets

File: src/Components/Navbar.js [hunk 6]
```diff
@@ -103,21 +94,36 @@ function Navbar() {
...
+            expand={expand}
+            closeExpand={closeExpand}
+            tooltip="Sponsor"
+            label="Sponsors"
+          />
           <NavItem
             icon={<FcCalendar />}
             route={"/Events"}
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:122]: doesn't need the brackets

**Review by ShawnCollinge — APPROVED** [review]
LGTM!

**Pull request merged**